static/js/message_view_header.js

```diff
@@ -98,9 +99,12 @@ function bind_title_area_handlers() {
         search.initiate_search();
         e.preventDefault();
         e.stopPropagation();
+        popovers.hide_all();
```

Comment:
stylistically, this is usually the first step in the click handler.

Reply:
Agreed, but as I saw more than half of the occurrences in the codebase are at the ending side of the handler. Updating the PR though.